content/departments/engineering/design/brand_guidelines/color.md

```diff
@@ -107,130 +107,200 @@ Primary logo colors have been marked. The secondary colors, along with shades an
+    </div>
     <div class="color color-sm" style="--color: #c7ffff">
       <code>#c7ffff</code>
-      <small>Blue Mist</small>
+      <small>Blue 200</small>
     </div>
     <div class="color color-sm" style="--color: var(--sg-sky-blue)">
       <code>#00cbec</code>
-      <small>Sky Blue</small>
+      <small>Blue 300</small>
       <div><img src="logo/versions/Sourcegraph_Logomark_WHT.svg" alt="Primary logo color" style="width: 1em; height: 1em"></div>
     </div>
     <div class="color color-sm" style="--color: #00a1c7; color: white">
       <code>#00a1c7</code>
-      <small>Pacific Blue</small>
+      <small>Blue 400</small>
     </div>
     <div class="color color-sm" style="--color: #005482; color: white">
       <code>#005482</code>
-      <small>Orient</small>
+      <small>Blue 500</small>
     </div>
   </div>
   <div class="color-palette-column">
+    <div class="color color-sm" style="--color: #d2fff1">
+      <code>#d2fff1</code>
+      <small>Green 100</small>
+    </div>
     <div class="color color-sm" style="--color: #c4ffe8">
       <code>#c4ffe8</code>
-      <small>Green Mist</small>
+      <small>Green 200</small>
     </div>
     <div class="color color-sm" style="--color: #8fedcf">
       <code>#8fedcf</code>
-      <small>Mint</small>
+      <small>Green 300</small>
     </div>
     <div class="color color-sm" style="--color: #17ab52; color: white">
       <code>#17ab52</code>
-      <small>Mountain Meadow</small>
+      <small>Green 400</small>
     </div>
     <div class="color color-sm" style="--color: #1f7d45; color: white">
       <code>#1f7d45</code>
-      <small>Eucalyptus</small>
+      <small>Green 500</small>
+    </div>
+  </div>
+  <div class="color-palette-column">
+    <div class="color color-sm" style="--color: #ffffff">
+      <code>#ffffff</code>
+      <small>White</small>
+    </div>
+    <div class="color color-sm" style="--color: #f9fafb">
+      <code>#f9fafb</code>
+      <small>Grayscale 50</small>
+    </div>
+    <div class="color color-sm" style="--color: #f1f1f1">
+      <code>#f1f1f1</code>
+      <small>Grayscale 100</small>
+    </div>
+    <div class="color color-sm" style="--color: #e7e7e7;">
+      <code>#e7e7e7</code>
+      <small>Grayscale 200</small>
+    </div>
+    <div class="color color-sm" style="--color: #c4c4c4;">
+      <code>#c4c4c4</code>
+      <small>Grayscale 300</small>
+    </div>
+    <div class="color color-sm" style="--color: #8a8a8a; color: white">
+      <code>#8a8a8a</code>
+      <small>Grayscale 400</small>
+    </div>
+    <div class="color color-sm" style="--color: #4d4d4d; color: white">
+      <code>#4d4d4d</code>
+      <small>Grayscale 500</small>
+    </div>
+    <div class="color color-sm" style="--color: #313131; color: white">
+      <code>#313131</code>
+      <small>Grayscale 600</small>
+    </div>
+    <div class="color color-sm" style="--color: #000000; color: white">
+      <code>#000000</code>
+      <small>Black</small>
     </div>
```

Comment:
Looks beautiful, thank you! Do you want the grayscale blocks to wrap like the colored ones are? If so, I'm happy to do that real quick @traceyljohnson 
![image](https://user-images.githubusercontent.com/59381432/177655385-7830d0aa-49cc-4e98-85b6-c1487e26b540.png)